object/slicer/options.go

```diff
@@ -0,0 +1,77 @@
+package slicer
+
+import (
+	"io"
+
+	"github.com/nspcc-dev/neofs-sdk-go/session"
+)
+
+// Options groups all options of SliceDataIntoObjects.
```

Comment:
`SliceDataIntoObjects` doesn't look like a `slicer` to me. It's a `slicer.SingleShot`, a real `slicer.Slicer` could take a session token, an object payload limit, a CID and an owner ID and then provide some simple
```
// maybe with neofs epoch parameter at the same time, but I'd not do that
func (s *Slicer) Slice(r io.Reader, attrs []string) (oid.ID, error)
```
which (`*Slicer`) will be reusable for some upload session which uses the same token for a set of objects.

Reply:
Another approach could be exposing the `io.Writer` interface, then the app will write to it in any way it wants to and all the magic will happen inside of this writer.

Reply:
So in your proposal `Slicer` is a reusable entity which is useful when application changes input data stream and, optionally, attributes.

The one-time write function will also be useful, for example, in the NeoFS CLI. It can be a helper func over `Slicer` which allows to not construct it explicitly.

So I suggest to support them both.

Reply:
Sure, both are useful in different contexts.